build/common/config/.golangci.yml

```diff
@@ -164,13 +180,43 @@ issues:
   # excluded by default patterns execute `golangci-lint run --help`
   exclude:
     - composite literal uses unkeyed fields
+    - return statements should not be cuddled if block has more than two lines
+    - declarations should never be cuddled
+    - don't use leading k in Go names
 
   exclude-rules:
-    # Exclude some linters from running on test files.
-    - path: _test\.go$|^tests/|^samples/
+    # Allow dot imports in the tests.
+    - path: _test\.go$|^test/
+      linters:
+        - gci
+        - golint
+        - revive
+        - stylecheck
+      source: \. "github\.com/onsi/(gomega|ginkgo)"
+    # Allow interface in the tests.
+    - path: _test\.go$|^test/
+      linters:
+        - forbidgo
+      source: fmt\.Print
+    # Allow returning interface in the tests.
+    - path: _test\.go$|^test/
+      linters:
+        - ireturn
+      source: func
+    # Add exceptions to API level tagliatelle violations. Can't use nolint comments since
+    # that affects the CRD descriptions of the fields.
+    - path: ^api/
       linters:
-        - errcheck
-        - maligned
+        - tagliatelle
+      source: json:"policy-templates|json:"extra_vars
```

Comment:
Could you please remove this since this check is specific to the policy propagator?